app-emacs/gcmh/gcmh-0.2.1.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+NEED_EMACS="24"
+
+inherit elisp
+
+DESCRIPTION="The Garbage Collector Magic Hack"
+HOMEPAGE="https://elpa.gnu.org/packages/gcmh.html"
```

Comment:
Would we want to include https://gitlab.com/koral/gcmh as the upstream/homepage as well?

Reply:
to be frank, I wasn't sure what the right choice is. there are multiple copies. I can include it. /shrug